BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -151,7 +154,9 @@ open class GiniBankNetworkingScreenApiCoordinator: GiniScreenAPICoordinator, Gin
         giniBankConfiguration = configuration
         giniBankConfiguration.documentService = documentService
         GiniBank.setConfiguration(configuration)
-
+        // TODO: No clientID user property for custom networking init
+        AnalyticsManager.userProperties = [.returnAssistantEnabled: configuration.returnAssistantEnabled,
```

Comment:
[suggestion]: I suggest having a separate method to set ` [.returnAssistantEnabled: configuration.returnAssistantEnabled,
                                           .returnReasonsEnabled: configuration.enableReturnReasons]` and reuse it.
 Maybe we would need to do same thing for client_id or not

Reply:
No. I was thinking to have a method here to set those properties and use it in both place: line 135 and here line 158.